lib/spack/spack/test/git_fetch.py

```diff
@@ -82,6 +83,8 @@ def test_bad_git(tmpdir, mock_bad_git):
             fetcher.fetch()
 
 
+@pytest.mark.skipif(sys.platform == 'win32',
+                    reason="Not supported on Windows (yet)")
```

Comment:
Could you re-check if these tests still fail on Windows? I think ideally we would at least ensure that URL and git-based fetching works on Windows, and I assume we have the necessary resources in the Windows testing environment (in this case: the Git program).

Reply:
Those that do not suffer lock failures pass. Presumably once I take care of the stage test lock the rest will pass. Will confirm once I can.